kiwi/system/setup.py

```diff
@@ -892,14 +908,18 @@ def _import_custom_scripts(self):
                 ]
             )
 
-    def _call_script(self, name):
+    def _call_script(self, name, option_list=None):
         script_path = os.path.join(self.root_dir, 'image', name)
         if os.path.exists(script_path):
+            options = option_list or []
             command = ['chroot', self.root_dir]
             if not Path.access(script_path, os.X_OK):
                 command += ['bash']
-            command += ['/image/' + name]
-            config_script = Command.call(command)
+            command += ['/image/' + name] + options
```

Comment:
This does work, yes, but I would suggest to clean this up and make it it less Perl-ish:
```python
command.append(os.path.join("/image", name))
command.extend(option_list)
```

Ideally, replace "/image" string with the constant variable.

Reply:
ok after reading more parts of that code I think it deserves some cleanup. The /image location imho deserves to be a default setting as it's also used at other classes and I will apply your suggestions to be less Perl-ish ... man I've not done perl for years ;)